### PR TITLE
add metrics on rent collection pieces

### PR DESCRIPTION
**Pull request by jeffwashington**
#### Summary of Changes

Fixes #

### CODE REVIEW

**Review by HaoranYi — DISMISSED** [review]
lgtm

File: runtime/src/bank.rs [hunk 1]
```diff
@@ -5222,33 +5237,49 @@ impl Bank {
...
+        metrics: &RentMetrics,
+    ) -> usize {
         let subrange = Self::pubkey_range_from_partition(partition);
 
+        let mut hold_range = Measure::start("hold_range");
         let thread_pool = &self.rc.accounts.accounts_db.thread_pool;
         self.rc
             .accounts
             .hold_range_in_memory(&subrange, true, thread_pool);
+        hold_range.stop();
```
Comment (apfitzge) [inline runtime/src/bank.rs:5253]: we aren't using the result of this measure, can we remove it or should it be added to the `RentMetrics`?

**Review by ckamm — APPROVED** [review]
remove/use hold_range measurement

File: runtime/src/bank.rs (hunk 1 above)
Comment (jeffwashington) [reply runtime/src/bank.rs:5255]: added

**Pull request merged**